tests/s2n_test.h

```diff
@@ -109,6 +109,8 @@ int test_count;
 #define EXPECT_NULL( ptr )      EXPECT_EQUAL( ptr, NULL )
 #define EXPECT_NOT_NULL( ptr )  EXPECT_NOT_EQUAL( ptr, NULL )
 
+#define EXPECT_LESS_THAN_EQUAL( p1, p2 ) EXPECT_TRUE( (p1) <= (p2) )
```

Comment:
Nit: I'm not sure this macro is any shorter / more readable than EXPECT_TRUE( p1 <= p2 ). Is it necessary?

Reply:
I added it because because we have a similar check_lte() check